pkgs/tools/networking/mpssh/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, openssh, which }:
+
+stdenv.mkDerivation rec {
+  pname = "mpssh";
+  version = "1.3.3";
+
+  src = fetchFromGitHub {
+    sha256 = "0iibzb9qrwf9wsmp2b8j76qw66ql8fw5jx02x82a22fsaznb5dng";
+    rev = "${version}";
+    repo = "${pname}";
+    owner = "ndenev";
+  };
+
+  buildInputs = [ openssh which];
```

Comment:
Would be better to override `SSHPATH` and `SCPPATH` explicitly instead of relying on `which`, no?